examples/log/humanoid_taichi.log

```diff
@@ -156,5 +156,5 @@ iter 	 cost 	      stop 	    grad 	  xreg 	      ureg 	 step 	 feas
  140  6.81843e+00  6.62800e-08  1.57323e-07  1.00000e-09  1.00000e-09   0.5000     1
 ('Finally reached = ', array([3.99963367e-01, 2.37599941e-05, 1.19998324e+00]))
 ('Distance between hand and target = ', 4.676946718296189e-05)
-('Distance to default state = ', 32.524482279937715)
```

Comment:
This is a bit weird. `Distance = 32.5`, and it is changed to `Distance = 1.6`? Were the logs wrong before?

Reply:
This is quiet weird indeed. I saw the same thing with codegen case. Do you think we should investigate this before merging it?

Reply:
I cannot explain you the reason of this difference, however, I can tell you that this, and other humanoid examples, does not use the impulse dynamics. Therefore, the changes propose in this PR should not affect them.

Additionally, I check qualitatively the resulted motion using Gepetto viewer and it seems to be the same.